### PR TITLE
Fix refresh access token request body

### PR DESCRIPTION
**Pull request by genzj**
Try fixing issue #28 by adding client ID and client secret to body of the refresh token request.

### CODE REVIEW

**Review by mnencia — APPROVED** [review]

**Comment by mnencia** [comment]
Good catch!

If you fix the linter warnings, I'll merge it ASAP.

File: mch/client.go [hunk 1]
```diff
@@ -43,6 +43,11 @@ type Client struct {
 	HTTPClient    http.Client    `json:"-"`
 }
 
+const (
+	clientId     = "9B0Gi617tROKHc2rS95sT1yJzR6MkQDm"
+	clientSecret = "oSJOB1KOWnLVZm11DVknu2wZkTj5AGKxcINEDtEUPE30jHKvEqorM8ocWbyo17Hd"
```
Comment (mnencia) [inline mch/client.go:48]:
```suggestion
	clientSecret = "oSJOB1KOWnLVZm11DVknu2wZkTj5AGKxcINEDtEUPE30jHKvEqorM8ocWbyo17Hd" // nolint:gosec
```

File: mch/client.go [hunk 2]
```diff
@@ -60,8 +65,8 @@ func Login(username string, password string) (*Client, error) {
 		"username":      username,
 		"password":      password,
 		"scope":         "openid offline_access nas_read_write nas_read_only user_read device_read",
-		"client_id":     "9B0Gi617tROKHc2rS95sT1yJzR6MkQDm",
-		"client_secret": "oSJOB1KOWnLVZm11DVknu2wZkTj5AGKxcINEDtEUPE30jHKvEqorM8ocWbyo17Hd",
+		"client_id":     clientId,
```
Comment (mnencia) [inline mch/client.go:68]:
```suggestion
		"client_id":     clientID,
```

File: mch/client.go [hunk 3]
```diff
@@ -105,7 +110,8 @@ func osType() string {
 func (c *Client) refreshAccessToken() error {
 	req := map[string]string{
 		"audience":      "mycloud.com",
-		"client_id":     c.Configuration.GetString("com.wd.portal", "portal.auth0.client"),
+		"client_id":     clientId,
```
Comment (mnencia) [inline mch/client.go:113]:
```suggestion
		"client_id":     clientID,
```

File: mch/client.go [hunk 4]
```diff
@@ -43,6 +43,11 @@ type Client struct {
 	HTTPClient    http.Client    `json:"-"`
 }
 
+const (
+	clientId     = "9B0Gi617tROKHc2rS95sT1yJzR6MkQDm"
```
Comment (mnencia) [inline mch/client.go:47]:
```suggestion
	clientID     = "9B0Gi617tROKHc2rS95sT1yJzR6MkQDm"
```

**Comment by genzj** [comment]
Just saw building error and your message. Many thanks for fixing the lint issues!

**Pull request merged**